CONTRIBUTING.md

```diff
@@ -150,8 +149,7 @@ Section](https://getkong.org/docs/latest/plugin-development/distribution/#distri
 of the Plugin Development Guide.
 
 To give visibility to your plugin, we advise that you post an announcement
-thread on the [mailing
-list](https://groups.google.com/forum/#!forum/konglayer). A good title for such
+thread on [Kong Nation](https://discuss.konghq.com/). A good title for such
```

Comment:
We probably need to update the announcement title since `[ANN]` is redundant when posting inside of an "Announcements" category :thinking:

Reply:
In fact, suggesting a title was only useful in the context of a mailing list (since it is hard to keep organized between questions, debates, and announcements. But in the context of a forum with a clearly defined "Announcements" category, not so much, it even feels pedantic... Let's remove this suggested title!